src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx

```diff
@@ -15,428 +15,108 @@ redirects:
 
 ## Benefits of OpenTelemetry [#benefits]
 
-OpenTelemetry provides a secure, vendor-neutral [specification](https://github.com/open-telemetry/opentelemetry-specification) for service and host instrumentation so that you can export data to distinct backends of your choice, such as New Relic. OpenTelemetry offers a single set of APIs and libraries that standardize how you collect and transfer telemetry data for your services and hosts.
-
-The following components make up the OpenTelemetry project:
+OpenTelemetry is a vendor-neutral open standard for instrumenting and exporting telemetry data. The project scope is quite extensive, including:
 
-* Specifications for the core pillars of observability to drive consistency across all projects. New Relic supports all of these [signals](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#signals): traces, metrics, and logs (see details [below](#how-it-works))
-* APIs that contain interfaces and implementations based on the specifications SDKs (reference implementations of the APIs) created specifically for languages like Java, Python, Go, Erlang, and more
-* Collectors that offer a vendor-agnostic implementation for processing and exporting
-* Exporters that enable you to send data to a backend of your choice
+* A [specification](https://opentelemetry.io/docs/specs/otel/) of language agnostic APIs for instrumenting the core pillars of observability (traces, metrics, logs), and SDKs for configuring how telemetry is exported out of process, with implementations exist in [11+](https://opentelemetry.io/docs/languages/) languages. There is a substantial catalog of instrumentation available built on top of these APIs.
+* [OTLP](https://opentelemetry.io/docs/specs/otlp/), a general-purpose telemetry data delivery protocol.
+* [Semantic conventions](https://opentelemetry.io/docs/specs/semconv/) describing the shape of telemetry data for common domains (i.e. HTTP servers, messaging systems, and many more).
+* The [collector](https://opentelemetry.io/docs/collector/), a highly configurable and extensible data collection and processing pipeline used to monitor infrastructure and enrich, filter, and otherwise transform telemetry.
 
-The components of OpenTelemetry work together to create some distinct advantages for capturing telemetry data:
+These components work together to create distinct advantages:
```

Comment:
I've rewritten the rows in this table with content I think better reflects the state of things today.